src/EventStore.Core/LogV3/LogV3SystemEventTypes.cs

```diff
@@ -0,0 +1,60 @@
+﻿using EventStore.Core.Services;
+using EventTypeId = System.UInt32;
+
+namespace EventStore.Core.LogV3
+{
+	public class LogV3SystemEventTypes {
+
+		public const EventTypeId FirstRealEventTypeNumber = 1024;
+		public const EventTypeId EventTypeInterval = 1;
+
+		public const EventTypeId EmptyEventTypeNumber = 0;
+		public const EventTypeId EventTypeDefinedNumber = 1;
+		public const EventTypeId StreamCreatedNumber = 2;
+		public const EventTypeId StreamMetadataNumber = 3;
+		public const EventTypeId StreamDeletedNumber = 4;
+		
+		public static bool TryGetSystemEventTypeId(string type, out EventTypeId eventTypeId) {
+			switch (type) {
+				case SystemEventTypes.EmptyEventType:
+					eventTypeId = EmptyEventTypeNumber;
+					return true;
+				case SystemEventTypes.EventTypeDefined:
+                	eventTypeId = EventTypeDefinedNumber;
```

Comment:
nit pick: indentation looks incorrect here (due to spaces)

Reply:
:+1: